src-tauri/src/main.rs

```diff
@@ -66,6 +66,9 @@ pub struct SystemStats {
     pub memory_used: u64,
     pub memory_free: u64,
     pub memory_cached: u64,
+    swap_space: u64,
+    swap_used: u64,
+    swap_free: u64,
```

Comment:
any reason why these are not public? if no need, then should we make all private?

Reply:
There was no reason to make em public as its a DTO (Data transfer object). But given that we will have tests down the round, i would like to suggest `pub(crate)` . 

For consistency , i have kept 'em public

Reply:
I dont really care, I just wanna have some consistency over coding style.